src/app/child-dev-project/notes/demo-data/notes_group-stories.ts

```diff
@@ -1,37 +1,40 @@
 import { WarningLevel } from "../../warning-level";
 
+/**
+ * Contains demo group notes. This data is used to create new Note instances with the provided values which then get saved to the MockDB.
+ */
 export const noteGroupStories = [
   {
-    category: "GUARDIAN_MEETING",
+    category: { name: "Guardians' Meeting", color: "#E1F5FE", isMeeting: true },
```

Comment:
because these objects are still transformed by the schema, I think just `category: { name: "Guardians' Meeting" }` should be sufficient and we can leave the color definition to the config file. Let's try that

Reply:
Just to clarify: Do you want me to rewrite the to-database-transformation so that this would work? As of now we check if the category object matches the (stringified) one in the config file (including color, isMeating properties) in the InteractionSchemaDatatype-class.